commands/operator-sdk/cmd/new.go

```diff
@@ -220,6 +235,19 @@ func doScaffold() {
 		log.Fatalf("failed to create %v: %v", rolePath, err)
 	}
 
+	// generate deploy/test-pod.yaml
+	testPodPath := filepath.Join(deployDir, "test-pod.yaml")
+	testPodfilegen := scaffold.NewTestPodCodegen(&scaffold.TestPodInput{})
```

Comment:
Pass the inputs for the test-pod.yaml template or else it won't get generated correctly
```Go
testPodfilegen := scaffold.NewTestPodCodegen(&scaffold.TestPodInput{
    ProjectName: filepath.Base(projectname), 
    Image: "OPERATOR_IMAGE",
    TestNamespaceEnv: test.TestNamespaceEnv,
})
```

Reply:
This actually shouldn't be in this file, only build, so I'll remove it from here